spec/jobs/sww/export/event_participations_export_job_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2025, Schweizer Wanderwege. This file is part of
+#  hitobito_sww and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sww.
+
+
+require "spec_helper"
+
+describe Sww::Export::EventParticipationsExportJob do
+  subject {
+    Export::EventParticipationsExportJob.new(format,
+      user.id,
+      event.id,
+      group.id,
+      params.merge(filename: filename))
+  }
+
+  let(:user) { people(:zuercher_leiter) }
+  let(:event) { events(:top_course) }
+  let(:group) { event.groups.first }
+  let(:filename) { AsyncDownloadFile.create_name("event_participation_export", user.id) }
+  let(:file) { AsyncDownloadFile.from_filename(filename, format) }
+
+  before do
+    SeedFu.quiet = true
+    SeedFu.seed [Rails.root.join("db", "seeds")]
```

Comment:
Hast du das von irgendwo her übernommen? Stört das nicht die anderen Tests, und dauert auch noch lange? Normalerweise arbeiten wir in den Specs mit den Fixtures und Fabricators soweit mir bewusst ist